src/Aks/Aks/help/Import-AzAksCredential.md

```diff
@@ -167,6 +167,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PublicFQDN
```

Comment:
The abbreviation FQDN is mostly spelled as "Fqdn" in Az. Is there special reasons why this one is not?